src/lib/formatDateClaim.js

```diff
@@ -89,7 +89,7 @@ function parseDate(wikidatatime, languageCode = "en") {
       return ordinalize(millenniumNumber) + " mill." + eraSuffix;
     case 7:
       let centuryIndex = Math.abs(Math.floor(year / 100));
-      let centuryNumber = year > 0 ? centuryIndex + 1 : centuryIndex;
+      let centuryNumber = centuryIndex; //year > 0 ? centuryIndex + 1 : centuryIndex;
```

Comment:
OK, would you be able to add a test in `formatDateClaim.test.js` with a 7 precision item?

Reply:
Yes I did.

Reply:
```suggestion
      let centuryNumber  = year > 0 ? Math.floor(year-1/ 100)) + 1 : Math.abs(Math.floor(year/ 100))
```

Reply:
done